lib/backy/railtie.rb

```diff
@@ -0,0 +1,7 @@
+module Backy
+  class Railtie < Rails::Railtie
+    rake_tasks do
```

Comment:
Re your commant about using it on flytaxien, the railtie is missing something in order to tie config from a Rails app to the gem. I tried to install the gem in the dummy app for the CLI branch here: https://github.com/rubynor/backy/pull/6/files#diff-ee0d7119520d0f50d8f99e8b73bc1707d5dbc978a549606f251e6a1067a98fbc
And it does not work. I guess we should focus on getting it to work for that dummy app first?

The railtie should have an `initializer` block like in Chippy
https://github.com/rubynor/chippy/blob/main/lib/chippy/client/railtie.rb

Reply:
Yeah i agree, will focus on getting it to work on the dummy app :+1: